src/rust/engine/process_execution/src/local.rs

```diff
@@ -593,15 +593,16 @@ pub trait CapturedWorkdir {
       }
       Err(msg) if msg == "deadline has elapsed" => {
         let stdout = Bytes::from(format!(
-          "Exceeded timeout of {:?} for local process execution, {}",
-          req.timeout, req.description
+          "Exceeded timeout of {:.1} seconds when executing local process: {}",
+          req.timeout.map(|dur| dur.as_secs_f32()).unwrap_or(-1.0),
+          req.description
         ));
         let stdout_digest = store.store_file_bytes(stdout.clone(), true).await?;
 
         Ok(FallibleProcessResultWithPlatform {
           stdout_digest,
           stderr_digest: hashing::EMPTY_DIGEST,
-          exit_code: -libc::SIGTERM,
+          exit_code: libc::SIGTERM,
```

Comment:
I believe the negative number here was intended to convey that the process was killed by a signal (which is somewhat of a UNIX convention). Making it a positive would imply that the process returned 15 as an exit code normally instead of exiting on a signal which would be technically incorrect. How do we exits via signals elsewhere when executing a `Process`?

Reply:
I see. I'll revert that change for the sake of this PR. I'm fine with an incremental improvement